docs/installation/presto-configuration.rst

```diff
@@ -3,21 +3,21 @@
 Configuring Presto
 ==================
 
-Presto has many configuration parameters that can be modified to
-tweak performance or add/remove features. While Presto should work decently well out-of-the-box,
+Presto configuration parameters can be modified to
+tweak performance or add/remove features. While Presto is designed to work well out-of-the-box,
 you still may need to make some changes.
 
 For example, it is often necessary to change the default memory configuration
-based on your cluster's capacity. The default max memory for each Presto server is 16GB, but in
-an environment with less than 16GB available per node, you will have to adjust the settings
-to be lower. Similarly, if you have a lot of memory (say, 120GB/node), you may want to allocate more
+based on your cluster's capacity. The default max memory for each Presto server
+is 16 GB, but if you have a lot of memory (say, 120GB/node), you may want to allocate more
 memory to Presto for better performance.
 
-In the second case, in order to update the max memory value to 60GB per node, 
-change the line in ``/etc/opt/prestoadmin/coordinator/jvm.config`` and
+In order to update the max memory value to 60 GB per node:
```

Comment:
I like this---it was the "in the second case" that was making me look for the first case, which wasn't labeled as the first case.